docs/release-procedure.md

```diff
@@ -25,7 +25,7 @@ Release per project:
 *   Upload to PyPI
 
         git clean -xfd
-        python setup.py sdist bdist_wheel --universal
+        python setup.py sdist bdist_wheel
```

Comment:
@jakirkham are there any other best practices that we should be doing here?

Reply:
We should start packaging the license file in wheels. ( https://github.com/dask/dask/pull/4988 )

Reply:
If we were still doing universal wheels, would have [moved this into `setup.cfg` like so]( https://github.com/pypa/sampleproject/blob/cbabca3f3f54a6e053e231d4cc89a46550049107/setup.cfg#L6-L13 ), but this isn't relevant any more.